Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM atlanticcrypto/cuda-ssh-server:10.2-cudnn
```

Comment:
Looking at this dockerfile, https://github.com/coreweave/cuda-ssh-server/blob/master/Dockerfile i can see that it isn't too complicated, you might want to consider just pasting that in here so the project isn't more associated with your current vendor than it needs to be.

Plus it looks like there might be some some default passwords exposed there.

Reply:
I guess we could do that at some point, maybe worth creating an issue for, but I don't think it's high priorty.

As for default passwords, actually, what we do is we disable authentication entirely (since it's all on a private network). Suboptimal, yes, but it works for now and we can figure it out later, concurrently with figuring out actual training.

Reply:
> Plus it looks like there might be some default passwords exposed there.

Not quite. When you create a deployment you can check the logs to see the password. After doing so, the logs are flushed.

Is it the most secure thing in the world? No
Is it a problem? I don't think so